src/styles/components/tile.scss

```diff
@@ -9,7 +9,7 @@ $size: 15rem;
   position: relative;
   background-color: $colour-uniprot-blue;
   display: inline-block;
-  margin: $global-margin;
+  margin: 0.5rem;
```

Comment:
Even if this is temporary, could we still use variables? in that case `$global-margin/2`